docs/deploystack.md

```diff
@@ -0,0 +1,10 @@
+## Install
+You can install this application using the `Open in Google Cloud Shell` button 
+below. 
+<!-- TODO: remove reference to the deploystack-enable branch when it pushes to main -->
+<a href="https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2Ftpryan%2Fmicroservices-demo&shellonly=true&cloudshell_image=gcr.io/ds-artifacts-cloudshell/deploystack_custom_image&cloudshell_git_branch=deploystack-enable" target="_new">
+    <img alt="Open in Cloud Shell" src="https://gstatic.com/cloudssh/images/open-btn.svg">
+</a>
```

Comment:
**Issue:**
I just tried this "Open in Google Cloud Shell" button.
It looks like DeployStack is still expecting a `.json` file — instead of `.yaml`.

I got the following error:
```
2022/10/14 15:35:47 could not read config file: unable to parse config file: config file not present, looking for deploystack.json or .deploystack/deploystack.json
```

Full logs:
```
Welcome to Cloud Shell! Type "help" to get started.
To set your Cloud Platform project in this session use “gcloud config set project [PROJECT_ID]”
nimjay@cloudshell:~$ cloudshell_open --repo_url "https://github.com/tpryan/microservices-demo" --page "shell" --git_branch "deploystack-enable" --force_new_clone
2022/10/14 15:35:45 Cloning https://github.com/tpryan/microservices-demo into /home/nimjay/cloudshell_open/microservices-demo
Cloning into '/home/nimjay/cloudshell_open/microservices-demo'...
remote: Enumerating objects: 7616, done.
remote: Counting objects: 100% (113/113), done.
remote: Compressing objects: 100% (80/80), done.
remote: Total 7616 (delta 58), reused 79 (delta 32), pack-reused 7503
Receiving objects: 100% (7616/7616), 30.38 MiB | 35.88 MiB/s, done.
Resolving deltas: 100% (5395/5395), done.
2022/10/14 15:35:47 Branch deploystack-enable already exists locally. Switching to that branch.
2022/10/14 15:35:47 could not read config file: unable to parse config file: config file not present, looking for deploystack.json or .deploystack/deploystack.json
2022/10/14 15:35:47 could not read config file: unable to parse config file: config file not present, looking for deploystack.json or .deploystack/deploystack.json

An error occured collecting the necessary information to run your Deploystack
application. Please try again.

If the problem persists, please file an issue with the Github repo:
https://github.com/tpryan/microservices-demo/issues
```

Reply:
Yeah it looks like that change has not be cached on the Cloud Shell master image yet. It can take a while. (week or two)

Reply:
I tried out the "Open in Google Cloud Shell" button today. It worked! 👏  :)